css/styles.css

```diff
@@ -77,3 +89,180 @@ body {
   padding: 0;
   margin: 0;
 }
+
+.works {
+  padding-inline: 24px;
+}
+
+.works-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  gap: 114px;
+  margin: 114px 0;
+  justify-content: center;
+}
+
+.card {
+  min-width: calc(327px - 16px);
+  min-height: calc(552px - 16px);
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.image-container {
+  width: 100%;
+  height: 220px;
+  border-radius: 8px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.image-container img {
+  max-width: 90%;
+}
+
+.info {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.info-title {
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  margin: 0;
+}
+
+.canopy {
+  display: flex;
+  font-size: 13px;
+  font-weight: 600;
+  color: #7a869a;
+  align-items: center;
+}
+
+.canopy ul {
+  display: flex;
+  gap: 25px;
+}
+
+.extract {
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.extract p {
+  margin: 0;
+}
+
+.technologies {
+  width: 100%;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 10px;
+  list-style: none;
+  color: #6070ff;
+}
+
+.technologies li {
+  background-color: #ebebff;
+  padding: 4px 12px;
+  border-radius: 8px;
+}
+
+.see {
```

Comment:
Kindly add hover and pressed styles to your "See Project" buttons. In this way you can show your pixel perfect implementation skill and also this states will increase the user experience of the web page.
<img width="460" alt="Screenshot 2023-03-24 at 11 02 32 PM" src="https://user-images.githubusercontent.com/25887253/227627721-a9bc7edd-db13-4bd8-a352-549cd89d239f.png">